opencti-platform/opencti-front/src/private/components/threats/threat_actors_group/ThreatActorGroupKnowledge.jsx

```diff
@@ -109,7 +109,6 @@ class ThreatActorGroupKnowledgeComponent extends Component {
                 entityLink={link}
                 defaultStartTime={threatActorGroup.first_seen}
                 defaultStopTime={threatActorGroup.last_seen}
-                allDirections={true}
```

Comment:
removing this is intentional ? you kept it in TAI

Reply:
Nop, my bad.